### PR TITLE
Display correct texts in Admin Panel

### PR DESCRIPTION
**Pull request by jacstn**
#### 1. Objective

This PR corrects some texts displayed in the Admin Panel.

#### 2. Description of change

Change texts in Admin Panel Omise XML definition file.

Enable/Disable -> Enabled/Disabled

#### 3. Quality assurance

Check if switch option is displayed correctly in Admin Panel

**🔧 Environments:**

- **Platform version**: Magento CE 2.1.5.
- **Omise plugin version**: Omise-Magento 2.1.
- **PHP version**: 7.0.16.

**✏️ Details:**

Example of displayed text in the admin panel.

Previously:
image

Currently
image

#### 5. Priority of change

Normal

#### 6. Additional Notes

N/A

### CODE REVIEW

File: etc/adminhtml/system.xml [hunk 1]
```diff
@@ -42,7 +42,7 @@
                     <label>Enable Credit Card Solution</label>
                     <comment>Powerful payment features that allows you to easily and securely accept credit/debit card payments on your store.</comment>
                     <field id="active" translate="label comment" type="select" sortOrder="90" showInDefault="1" showInWebsite="1" showInStore="1">
-                        <label>Enable/Disable</label>
+                        <label>Enabled/Disabled</label>
```
Comment (jonrandy) [inline etc/adminhtml/system.xml]: This should say `Enabled?` as I suggested when I previously mentioned it. This still doesn't make sense

Comment (jacstn) [reply etc/adminhtml/system.xml]: @jonrandy why as English native speaker you just won't simply say what should be here? Whatever you say, I will add it here.

Comment (jonrandy) [reply etc/adminhtml/system.xml]: I have said. Twice.

`Enabled?`

Comment (jacstn) [reply etc/adminhtml/system.xml]: sorry.. I misunderstood. 
I changed it.

**Comment by jacstn** [comment]
to easy see changes: https://github.com/omise/omise-magento/compare/2-citi-points...2-admin-panel-better-texts

File: etc/adminhtml/system.xml [hunk 2]
```diff
@@ -39,11 +39,11 @@
...
 
                 <group id="omise_cc" translate="label" sortOrder="80" showInDefault="1" showInWebsite="1" showInStore="1">
-                    <label>Enable Credit Card Solution</label>
+                    <label>Enable Credit Card Payments</label>
                     <comment>Powerful payment features that allows you to easily and securely accept credit/debit card payments on your store.</comment>
                     <field id="active" translate="label comment" type="select" sortOrder="90" showInDefault="1" showInWebsite="1" showInStore="1">
-                        <label>Enable/Disable</label>
-                        <comment>Enable Omise Credit Card Solution.</comment>
+                        <label>Enabled?</label>
+                        <comment>Enable Omise Credit Card Payments.</comment>
```
Comment (jonrandy) [inline etc/adminhtml/system.xml]: Do we need 'Omise'? This will appear within the settings for the Omise plugin anyway

Comment (jonrandy) [reply etc/adminhtml/system.xml]: 'Omise' doesn't appear in most of the other methods below

Comment (jacstn) [reply etc/adminhtml/system.xml]: I removed it.

**Comment by jacstn** [comment]
@jonrandy I think it is ready to approve.

File: etc/adminhtml/system.xml [hunk 3]
```diff
@@ -94,12 +94,27 @@
...
+                    <comment>Enables customers to pay using Citi Points.</comment>
+                    <field id="active" translate="label comment" type="select" sortOrder="170" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Enabled?</label>
+                        <comment>Enable payment with Citi Points.</comment>
+                        <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                        <config_path>payment/omise_offsite_citipoints/active</config_path>
+                    </field>
+                    <field id="title" translate="label" type="text" sortOrder="180" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Title</label>
+                        <comment>This controls the title which the user sees during checkout.</comment>
```
Comment (jonrandy) [inline etc/adminhtml/system.xml]: The word 'which' is not needed here (and on the other titles)

File: etc/adminhtml/system.xml [hunk 4]
```diff
@@ -110,11 +125,11 @@
                     </field>
                 </group>
                 <group id="omise_offsite_installment" translate="label" sortOrder="220" showInDefault="1" showInWebsite="1" showInStore="1">
-                    <label>Enable Installment Solution</label>
+                    <label>Enable Installment Payments</label>
                     <comment>Enables customers to pay using Installment method</comment>
```
Comment (jonrandy) [inline etc/adminhtml/system.xml]: Enables customers to pay in installments

File: etc/adminhtml/system.xml [hunk 5]
```diff
@@ -125,11 +140,11 @@
                     </field>
                 </group>
                 <group id="omise_offsite_truemoney" translate="label" sortOrder="260" showInDefault="1" showInWebsite="1" showInStore="1">
-                    <label>Enable True Money Solution</label>
+                    <label>Enable True Money Payments</label>
                     <comment>Enables customers to pay using True Money method</comment>
```
Comment (jonrandy) [inline etc/adminhtml/system.xml]: 'method' is unnecessary here

File: etc/adminhtml/system.xml [hunk 6]
```diff
@@ -140,11 +155,11 @@
                     </field>
                 </group>
                 <group id="omise_offline_conveniencestore" translate="label" sortOrder="290" showInDefault="1" showInWebsite="1" showInStore="1">
-                    <label>Enable Convenience Store Payment</label>
+                    <label>Enable Convenience Store Payments</label>
                     <comment>Enables customers to easily pay using Convenience Store method (only available in Japan).</comment>
```
Comment (jonrandy) [inline etc/adminhtml/system.xml]: 'payment' instead of 'method' would be better here

File: etc/adminhtml/system.xml [hunk 7]
```diff
@@ -65,92 +65,107 @@
...
                         <label>Title</label>
-                        <comment>This controls the title which the user sees during checkout.</comment>
+                        <comment>This controls the title the user sees during checkout.</comment>
                         <config_path>payment/omise_offsite_internetbanking/title</config_path>
                     </field>
                 </group>
                 <group id="omise_offsite_alipay" translate="label" sortOrder="160" showInDefault="1" showInWebsite="1" showInStore="1">
-                    <label>Enable Alipay Solution</label>
+                    <label>Enable Alipay Payments</label>
                     <comment>Enables customers to pay using Alipay method</comment>
```
Comment (jonrandy) [inline etc/adminhtml/system.xml]: Method is unnecessary here

File: etc/adminhtml/system.xml [hunk 8]
```diff
@@ -65,92 +65,107 @@
...
-                        <comment>This controls the title which the user sees during checkout.</comment>
+                        <comment>This controls the title the user sees during checkout.</comment>
                         <config_path>payment/omise_offsite_internetbanking/title</config_path>
                     </field>
                 </group>
                 <group id="omise_offsite_alipay" translate="label" sortOrder="160" showInDefault="1" showInWebsite="1" showInStore="1">
-                    <label>Enable Alipay Solution</label>
-                    <comment>Enables customers to pay using Alipay method</comment>
+                    <label>Enable Alipay Payments</label>
+                    <comment>Enables customers to pay using Alipay</comment>
```
Comment (jonrandy) [inline etc/adminhtml/system.xml]: Full stop needed for consistency with other comments

Comment (jonrandy) [inline etc/adminhtml/system.xml]: Full stop needed for consitency with other payment methods

File: etc/adminhtml/system.xml [hunk 9]
```diff
@@ -65,92 +65,107 @@
...
                     <field id="title" translate="label" type="text" sortOrder="180" showInDefault="1" showInWebsite="1" showInStore="1">
                         <label>Title</label>
-                        <comment>This controls the title which the user sees during checkout.</comment>
+                        <comment>This controls the title the user sees during checkout.</comment>
                         <config_path>payment/omise_offsite_alipay/title</config_path>
                     </field>
                 </group>
+                <group id="omise_offsite_citipoints" translate="label" sortOrder="160" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Enable payment with Citi Points</label>
+                    <comment>Enables customers to pay using Citi Points</comment>
```
Comment (jonrandy) [inline etc/adminhtml/system.xml]: Full stop

File: etc/adminhtml/system.xml [hunk 10]
```diff
@@ -65,92 +65,107 @@
...
-                        <comment>This controls the title which the user sees during checkout.</comment>
+                        <comment>This controls the title the user sees during checkout.</comment>
                         <config_path>payment/omise_offline_tesco/title</config_path>
                     </field>
                 </group>
                 <group id="omise_offsite_installment" translate="label" sortOrder="220" showInDefault="1" showInWebsite="1" showInStore="1">
-                    <label>Enable Installment Solution</label>
-                    <comment>Enables customers to pay using Installment method</comment>
+                    <label>Enable Installment Payments</label>
+                    <comment>Enables customers to pay in installments</comment>
```
Comment (jonrandy) [inline etc/adminhtml/system.xml]: Full stop

File: etc/adminhtml/system.xml [hunk 11]
```diff
@@ -65,92 +65,107 @@
...
-                        <comment>This controls the title which the user sees during checkout.</comment>
+                        <comment>This controls the title the user sees during checkout.</comment>
                         <config_path>payment/omise_offsite_installment/title</config_path>
                     </field>
                 </group>
                 <group id="omise_offsite_truemoney" translate="label" sortOrder="260" showInDefault="1" showInWebsite="1" showInStore="1">
-                    <label>Enable True Money Solution</label>
-                    <comment>Enables customers to pay using True Money method</comment>
+                    <label>Enable True Money Payments</label>
+                    <comment>Enables customers to pay using True Money</comment>
```
Comment (jonrandy) [inline etc/adminhtml/system.xml]: Full stop

**Comment by guzzilar** [comment]
Is the latest commit addressing all the feedbacks already? Shall we merge this?
cc @jonrandy

**Comment by aashishgurung** [comment]
Stale PR.

**Pull request closed without merging**